build_releases

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -eu
+
+export GO111MODULE=on
+export GOFLAGS=-mod=readonly
+export CGO_ENABLED=0
+
+eval $(go env)
+if [ -z ${BIN_PATH+a} ]; then
+	export BIN_PATH=${PWD}/bin/releases/
+fi
+
+build_release() {
+    export NAME="fcct-${1}"
+    echo "building ${NAME}"
+    go build -o ${BIN_PATH}/${NAME} internal/main.go
+}
+
+export GOOS=linux
+export GOARCH=amd64
+build_release x86_64-unknown-linux-gnu
```

Comment:
Minor/optional: could just make `build_release` take the OS and arch as paramters.

Reply:
eh, I'll leave it off until it gets hairy. This matches the Ignition one too.